components/modal/modal.html

```diff
@@ -0,0 +1,80 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Button | Scale UI</title>
+    <link rel="stylesheet" href="../../css/component.css">
+    <link rel="stylesheet" href="modal.css">
+</head>
+<body>
+    <div class="container">
+        <!-- navbar -->
+        <nav class="navbar">
+            <div class="left-nav">
+                <div class="menu-icon">
+                    <i class="fas fa-bars"></i>
+                </div>
+                <div>
+                    <i class="fas fa-strikethrough"></i>
+                    <a href="../../index.html">Scale UI</a>
+                </div>
+            </div>
+            <ul class="right-nav">
+                <li><a href="https://github.com/anshulraheja/" target="_blank"><i class="fab fa-github-square"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/anshul-raheja/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+            </ul>
+        </nav>
+        <!-- content -->
+        <section class="main-section">
+            <!-- sidebar -->
+            <div class="sidebar-container">
+                <div class="introduction-list">
+                    <h3>Get Started</h3>
+                    <ul>
+                        <li><a href="../main.html">Installation</a></li>
+                    </ul>
+                </div>
+                <div class="component-list">
+                    <h3>Components</h3>
+                    <ul>
+                        <li><a href="../alert/alert.html">Alert</a></li>
+                        <li><a href="../avatar/avatar.html">Avatar</a></li>
+                        <li><a href="../badge/badge.html">Badge</a></li>
+                        <li><a href="../button/button.html">Button</a></li>
+                        <li><a href="../card/card.html">Card</a></li>
+                        <li><a href="../image/image.html">Image</a></li>
+                        <li><a href="../input/input.html">Input</a></li>
+                        <li><a href="../lists/lists.html">Lists</a></li>
+                        <li><a href="../modal/modal.html" class="active-page">Modal</a></li>
+                        <li><a href="../navigation/navigation.html">Navigation</a></li>
+                        <li><a href="../rating/rating.html">Rating </a></li>
+                        <li><a href="../grid/grid.html">Responsive Grid</a></li>
+                        <li><a href="../slider/slider.html">Slider</a></li>
+                        <li><a href="../text-utilities/textUtilities.html">Text Utilities</a></li>
+                        <li><a href="../toast/toast.html">Toast</a></li>
+                    </ul>
+                </div>
+            </div>
+            <!-- documentation -->
+            <div class="documentation-container">
+                <h1>Modal</h1>          
+                <div class="modal">
```

Comment:
Suggestion: A bit of margin between these two elements in your documentation would look good. 
The rest looks good to merge.